### ISSUE TITLE
Training loss: 0.000000；SDR value remains unchanged.

### ISSUE THREAD

**Issue by JuYangFu** [ZFTurbo/Music-Source-Separation-Training#30]
Hello, author, why is the SDR the same for every epoch after 60 epochs of training using the musdb18hq dataset, and the training loss remains 0 all the time? I used the following command:

**Comment by JuYangFu** [comment]
python train.py --model_type htdemucs --config_path configs/config_musdb18_htdemucs.yaml  --results_path results/model1.ckpt  --data_path '/data2/fjy/demucs-main/musdbhq/train' --valid_path /data2/fjy/demucs-main/musdbhq/test --device_ids 0
![image](https://github.com/ZFTurbo/Music-Source-Separation-Training/assets/133643578/5ac04597-e5f8-478a-84e5-0a58c61ef774)

**Comment by ZFTurbo** [comment]
What is your batch_size for training?

**Comment by JuYangFu** [comment]
My batch_size is 1
![image](https://github.com/ZFTurbo/Music-Source-Separation-Training/assets/133643578/2f695688-3c8b-4743-b78e-81cd14460646)

**Comment by ZFTurbo** [comment]
It's not possible for current loss functin. Batch size must be at least 2.

**Comment by JuYangFu** [comment]
Sorry,this file
![image](https://github.com/ZFTurbo/Music-Source-Separation-Training/assets/133643578/61485531-0a56-46da-bca2-63857341378e)

**Comment by JuYangFu** [comment]
Should I simply set the batch_size to 2?

**Comment by ZFTurbo** [comment]
> Should I simply set the batch_size to 2?

It's better to find maximum which still fit in your GPU memory

**Comment by JuYangFu** [comment]
How can I use two GPUs for training?

**Comment by ZFTurbo** [comment]
Add parameter in comand line: `--device_ids 0 1 2 3` - and list number of GPUs you want to use.

**Comment by JuYangFu** [comment]
Thank you!!

**Comment by idkcode22** [comment]
> It's not possible for current loss functin. Batch size must be at least 2. And you show not correct file on screenshot )

Can you add in readme that batch size has to be more than 2 to properly train?
